models/user.js

```diff
@@ -0,0 +1,49 @@
+var mongoose = require('mongoose')
+  , Schema = mongoose.Schema
+  , passportLocalMongoose = require('passport-local-mongoose');
+
+var UserSchema = new Schema({
+  username: {
+    type: String
+    , required: true
+    , lowercase: true // 'ABC' is equivalent 'abc'
+    , trim: true // ' abc ' will be trimmed to 'abc'
+    , index: { unique: true }
+  },
+  password: {
+    type: String
+  },
+  date_created: { // when the user created his account
+    type: Date
+    , required: true
+    , default: Date.now
+  },
+  banned: { // whether the user is banned by admin
+    type: Boolean
+    , required: true
+    , default: false
+  },
+  major: { // the user's academic major
+    type: String
+    , trim: true
+  },
+  is_admin: {
+    type: Boolean
+    , required: true
+    , default: false
+  },
+  locked: { // locked because they incorrectly loogged in 3 times
+    type: Boolean
+    , required: true
+    , default: false
+  },
+  bio: { // This is a free text for a user to talk about their interests
+    type: String
+    , trim: true
+  }
```

Comment:
Yes. You first need to enable `limitAttempts`, and add this `attempts` field to your schema. I don't think `passport-local` can change the schema for you.